JS101/running-javascript.md

```diff
@@ -22,6 +26,46 @@ Once the console is open, writing code is as easy as typing it in.  To get multi
 
 In many cases, if you just want to quickly try something out, using the browser console is incredibly quick and easy.
 
+__Word of Caution__:  
```

Comment:
at this point, students should not be expected to know anything about functions, or the difference between returning and logging.